client/src/main/java/io/getstream/chat/android/client/call/Call.kt

```diff
@@ -5,18 +5,19 @@ import androidx.annotation.WorkerThread
 import io.getstream.chat.android.client.errors.ChatError
 import io.getstream.chat.android.client.utils.Result
 
-interface Call<T> {
+public interface Call<T> {
 
     @WorkerThread
-    fun execute(): Result<T>
+    public fun execute(): Result<T>
 
     @UiThread
-    fun enqueue(callback: (Result<T>) -> Unit = {})
+    public fun enqueue(callback: (Result<T>) -> Unit = {})
 
-    fun cancel()
-    fun <K> map(mapper: (T) -> K): Call<K>
-    fun onError(handler: (ChatError) -> Unit): Call<T>
-    fun onSuccess(handler: (T) -> Unit): Call<T>
-    fun <K> zipWith(call: Call<K>): Call<Pair<T, K>>
-    fun <C, B> zipWith(callK: Call<C>, callP: Call<B>): Call<Triple<T, C, B>>
+    public fun cancel()
+
+    public fun <K> map(mapper: (T) -> K): Call<K>
+    public fun onError(handler: (ChatError) -> Unit): Call<T>
+    public fun onSuccess(handler: (T) -> Unit): Call<T>
+    public fun <K> zipWith(call: Call<K>): Call<Pair<T, K>>
+    public fun <C, B> zipWith(callK: Call<C>, callP: Call<B>): Call<Triple<T, C, B>>
```

Comment:
Are these methods supposed to be public API, or are they just for our own internal use?